src/foundry/common/prosemirror/dropdown.d.mts

```diff
@@ -18,6 +18,9 @@ declare class ProseMirrorDropDown {
    */
   constructor(title: string, items: ProseMirrorDropDownMenu.Entry[], options?: Options);
 
+  // placeholder private attribute to support subclasses
```

Comment:
Nit; the comment would be more accurate as "to limit what can count as a subtype at compile time".